docs/website/tds/reference/ReplayLogs.adoc

```diff
@@ -0,0 +1,72 @@
+:source-highlighter: coderay
+:linkcss:
+:stylesheet: ../tds.css
+
+= Testing a new version of the TDS
+
+Upgrading to a new version of the TDS can be a daunting task, especially when trying to figure out if your data collections are still accessible as your users expect.
+Perhaps one of the best "tests" a TDS administrator can do, although certainly no 100% complete, is to replay old user requests against a new, parallel test server to assess any issues. Note that the overhead of running a parallel test instance of Tomcat should be quite small, especially if the server is restricted to internal users only.
```

Comment:
got it - thanks @dopplershift!

On Tue, Feb 9, 2016 at 4:00 PM, Ryan May <notifications@github.com> wrote:

> In docs/website/tds/reference/ReplayLogs.adoc
> <https://github.com/Unidata/thredds/pull/416#discussion_r52389110>:
>
> > @@ -0,0 +1,72 @@
> > +:source-highlighter: coderay
> > +:linkcss:
> > +:stylesheet: ../tds.css
> > +
> > += Testing a new version of the TDS
> > +
> > +Upgrading to a new version of the TDS can be a daunting task, especially when trying to figure out if your data collections are still accessible as your users expect.
> > +Perhaps one of the best "tests" a TDS administrator can do, although certainly no 100% complete, is to replay old user requests against a new, parallel test server to assess any issues. Note that the overhead of running a parallel test instance of Tomcat should be quite small, especially if the server is restricted to internal users only.
>
> no -> not
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Unidata/thredds/pull/416/files#r52389110>.
>